harness/Makefile

```
@@ -0,0 +1,204 @@
+	  || { echo "    engine not reachable — run \`make engine\` first (or use \`make all\`)"; exit 1; }
+	@echo "==> spawning $(words $(WORKERS)) dep workers..."
+	@for w in $(WORKERS); do $(MAKE) --no-print-directory _spawn-one W=$$w; sleep 0.1; done
+	@echo "==> spawning harness..."
+	@$(MAKE) --no-print-directory _spawn-one W=harness
+	@echo "==> waiting 2s for registrations to settle..."
+	@sleep 2
+	@echo "==> done. PIDs in $(PIDS_DIR)  Logs in $(LOGS_DIR)"
+
+# Internal: spawn a single worker by name. Caller passes W=<worker>.
+.PHONY: _spawn-one
+_spawn-one:
```

checkmake (0.3.2)

[warning] Target body for "_spawn-one" exceeds allowed length of 5 lines (23).

(maxbodylength)